modules/json_content/json_content.module

```diff
@@ -1,10 +1,57 @@
 <?php
 
+use Drupal\Core\Entity\EntityInterface;
+use Drupal\json_content\EntityJsonTask;
+
 /**
  * Implements hook_toolbar().
  */
 function json_content_toolbar_alter(array &$items) {
   // The front page redirects to admin/content and content is generally not
   // viewed outside of the API so this link makes no sense.
   unset($items['home']);
-}
\ No newline at end of file
+}
+
+/**
+ * Implements hook_entity_operation_alter().
+ */
+function json_content_entity_operation_alter(array &$operations, EntityInterface $entity) {
+  if (isset($operations['edit'])) {
+    $operations['edit']['attributes']['title'] = t('Edit "@title"', [
```

Comment:
Can we just drop the quotes from around title? They end up getting escaped so they look goofy in the inspector and and hard to read/target in source.